server/utils/utils.ts

```diff
@@ -72,3 +73,23 @@ export const removeBlankSummaryListItems = (items: Array<SummaryListItem>): Arra
     return false
   })
 }
+
+export const mapApiPersonRiskForUi = (risks: PersonRisks): PersonRisksUI => {
```

Comment:
I see why you've not tested this, but it might be worth doing it, just so we're documenting the functionality

Reply:
Yeah no probs!